Directory.Build.props

```diff
@@ -46,23 +46,27 @@
 
   <!-- The TFMs to build and test against. -->
   <PropertyGroup>
-    <NetCoreAppCurrentVersion>7.0</NetCoreAppCurrentVersion>
+    <NetCoreAppCurrentVersion>8.0</NetCoreAppCurrentVersion>
```

Comment:
Shouldn't we be using the values we have in eng/Versions.props that we added in this PR? https://github.com/dotnet/runtime/pull/78354/commits/17764b999e7dfece2e72ca1bf7c68aa73465c8cd

Here, and also elsewhere where we hardcode the minor or major numbers.

Reply:
Directory.Build.props has access to Versions.props but we don't want to use the version properties from there as that would complicate branding to .NET 9, as it would mean that the package version branding change would also rev the NetCoreAppCurrent version, and we don't want that.